ballerina/Ballerina.toml

```diff
@@ -5,6 +5,7 @@ version = "2.2.0"
 authors = ["Ballerina"]
 keywords = ["http", "network", "service", "listener", "client"]
 repository = "https://github.com/ballerina-platform/module-ballerina-http"
+icon = "icon.png"
```

Comment:
Shouldn't this be added to the Ballerina.toml template under build-config?

Reply:
Added